Source/WebKit/UIProcess/ios/fullscreen/WKFullscreenStackView.mm

```diff
@@ -56,25 +61,35 @@ - (instancetype)init
         return nil;
 
     [self setClipsToBounds:YES];
-
+#if PLATFORM(APPLETV)
+    _backgroundView = adoptNS([[UIView alloc] initWithFrame:frame]);
+    _backgroundView.get().backgroundColor = [UIColor blackColor];
+#else
     _backgroundView = adoptNS([allocAVBackgroundViewInstance() initWithFrame:frame]);
-    [self addSubview:_backgroundView.get()];
-
     // FIXME: remove this once AVBackgroundView handles this. https://bugs.webkit.org/show_bug.cgi?id=188022
     [_backgroundView setClipsToBounds:YES];
 ALLOW_DEPRECATED_DECLARATIONS_BEGIN
     [_backgroundView.get().layer setContinuousCorners:YES];
 ALLOW_DEPRECATED_DECLARATIONS_END
     [_backgroundView.get().layer setCornerRadius:16];
-
+#endif
+    [self addSubview:_backgroundView.get()];
     return self;
 }
 
-- (void)addArrangedSubview:(UIView *)subview applyingMaterialStyle:(AVBackgroundViewMaterialStyle)materialStyle tintEffectStyle:(AVBackgroundViewTintEffectStyle)tintEffectStyle
+#if PLATFORM(APPLETV)
+- (void)addArrangedSubviewForTV:(UIView *)subview
+{
+    [_backgroundView.get() addSubview:subview];
+    [self addArrangedSubview:subview];
+}
+#else
+- (void)addArrangedSubviewaddArrangedSubview:(UIView *)subview applyingMaterialStyle:(AVBackgroundViewMaterialStyle)materialStyle tintEffectStyle:(AVBackgroundViewTintEffectStyle)tintEffectStyle
```

Comment:
should be `addArrangedSubview` instead of `addArrangedSubviewaddArrangedSubview`

Reply:
Lol thanks! Sloppy proofread